ui/.env.development

```diff
@@ -1,6 +1,6 @@
-VITE_DATASET_COUNT_BINS=1,2,3,4,5
+VITE_DATASET_COUNT_BINS=2,3,4,5,6
```

Comment:
Piggybacking a change on dev environments - lightest color isn't being triggered on tiles with a dataset count of 1